lib/commands/exam.js

```diff
@@ -35,11 +35,10 @@ module.exports = TestCommand.extend({
       description: 'Runs your split tests on parallel child processes.'
     },
     {
-      name: 'load-balance',
-      type: Boolean,
-      default: false,
+      name: 'browser-count',
```

Comment:
`browser-count` does not, for me anyways, imply load balancing. I wonder if we can do anything about that?